app/BUILD.bazel

```diff
@@ -406,6 +406,10 @@ in order to keep Gradle building.
      """,
 ) for file in BINDING_ADAPTERS_WITH_RESOURCE_IMPORTS]
 
+# Snap Helper
```

Comment:
Nit: comment isn't really needed--let's remove it. The code is self-documenting here.

Reply:
Done